doc/yagpdbcc.txt

```diff
@@ -0,0 +1,150 @@
+*yagpdbcc.txt* *yagpdbcc*
+
+YAGPDB Custom Commands -- Intro
+
+File: yagpdbcc.txt
+Version: 1.0
+Maintainer: Luca Zeuch <l-zeuch@email.de>
+Description: YAGPDB Custom Commands Vim Plugin
+
+License:
+    Copyright (C) 2021    Lucas Ritzdorf, Luca Zeuch
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+CONTENTS                                                   *yagpdbcc-contents*
+============================================================================
+1.  Overview ............................... |yagpdbcc-overview|
+2.  Installation ........................... |yagpdbcc-install|
+3.  Commands ............................... |yagpdbcc-commands|
+4.  Documentation .......................... |yagpdbcc-documentation|
+    4.1  Introduction ...................... |yagpdbcc-intro|
+    4.2  Templates ......................... |yagpdbcc-templates|
+                                             |yagpdbcc-functions|
+
+OVERVIEW                                                   *yagpdbcc-overview*
+============================================================================
+This is the YAGPDB Custom Commands (CC) plugin for Vim and Neovim.
+It provides syntax highlighting as well as bare boned documentation
+for the templates and structures in use.
+
+Useful Links~
```

Comment:
I'm probably going to change this format across all files to UPPERCASE instead, it looks quite weird in Vim itself.
```suggestion
USEFUL LINKS
```
Let me know what you think, both approaches have their benefit -- the first one allows us to have title and tag on the same line, the second approach gives the chapter titles some sense of hierarchy.

Reply:
After sleeping over it for a few days, I'll keep it as is.